src/cross.js

```diff
@@ -306,6 +332,7 @@ export const accountDataToAccount = ({
     index,
     freshAddress,
     freshAddressPath,
+    swapHistory: swapHistory.map(fromSwapOperationRaw),
```

Comment:
Can you run a small study on how much time it do to import accounts if you have let's say 50 swap history? (You can fake data just to look at how much qr code it generates)